### PR TITLE
[CCE] add CCE ICAgent annotation documentation

### PR DESCRIPTION
**Pull request by AdamVB**
## Summary of the Pull Request
Add documentation on how to install ICAgent automatically on CCE cluster creation

### CODE REVIEW

**Comment by AdamVB** [comment]
This takes care of #1534 and #1285

**Comment by artem-lifshits** [comment]
Hello @AdamVB labeler check is failing because you're trying to merge commit from local branch (AdamVB:devel).
I can recreate PR if it's ok for you.

**Comment by AdamVB** [comment]
@artem-lifshits that would be great. Thx!

File: docs/resources/cce_cluster_v3.md [hunk 1]
```diff
@@ -37,6 +37,33 @@ resource "opentelekomcloud_cce_cluster_v3" "cluster_1" {
...
+  description = "Create cluster"
+
+  cluster_type           = "VirtualMachine"
+  flavor_id              = var.flavor_id
+  vpc_id                 = var.vpc_id
+  subnet_id              = var.subnet_id
+  container_network_type = "overlay_l2"
+  authentication_mode    = "rbac"
+  kube_proxy_mode        = "ipvs"
+  annotations = { "cluster.install.addons.external/install" = "[{\"addonTemplateName\":\"icagent\"}]"  }
```
Comment (lego963) [inline docs/resources/cce_cluster_v3.md:63]: format pls

**Review by lego963 — CHANGES_REQUESTED** [review]

**Comment by artem-lifshits** [comment]
@AdamVB documentation was updated in #1926 , closing this PR.

**Pull request closed without merging**